### ISSUE TITLE
Game flow

### ISSUE THREAD

**Issue by Mephistophelesss** [island-troll-tribes/island-troll-tribes#448]
After playing public games, I've realized that non-competitive game have a very clunky game flow. Not having objectives is a really bad thing for a game and currently we have only one. It's killing the other team but it doesn't really have any evident path to follow (do this and win recipe). Giving new players a lot of freedom is cool but when you give them too much and they just lose sight ( Ex: "What now?"* )

I think the current meta of ITT doesn't seem very inciting for new players. The first 15 mins ( let's call it the gathering period ) is where experienced players will distinguish themselves from and new players (better at hunting, crafting, buying from boat and good item gathering). At a competitive level, gathering period lasts around 15-30 mins (because we know the meta), however for new player gathering period never really starts or end ( new players will tend to wander aimlessly, "Then what?"* ). Besides, ITT game flow is very slow, you have to put in work to craft the gear that will activate the PvP period,  compared to fast spaced games with objectives.

My suggestion would be to add more objectives (one during respawn time, another during early game, and more as the game keeps going). These objectives should have good rewards so that it force both team to fight over it (dropping spirit ward, or other niche item that can secure the win). 

Accordingly I think roles that suffer from early phase should bring benefit to the team in compensation. ( Ex: gatherer spawns with forge, scout spawns with workshop ).

Objectives don't have to be bosses! It can be securing a certain zone or bringing a certain item somewhere, just need clear objectives then player will have something they know they should be doing. We just need to slowly show them the way that leads to fighting and killing the opposing tribe.

Also i think we should add a fast crafting mode, where we reduce the mats needed to craft items. Mode would make the game less about gathering and more about fighting.

TL;DR : Add objectives throughout the game, compensate boring late game roles, fast crafting mode.

**Comment by Jaccouille** [comment]
Good point about new player who doesn't now what to do.

Where do you think those objectives should be located on the map? Last year I thought about reworking the aligator man on the edge of the map, make them a real boss fight with a few abilities to dodge and a kind of unique reward, however, I felt like pushing the player to go on the edge of the map to kill those miniboss make them loose control of the middle island with mammoth, so I kinda gave up.
